session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -913,6 +914,38 @@ public SessionDataSet executeQueryStatement(String sql)
         execResp.isIgnoreTimeStamp());
   }
 
+  /**
+   * query eg. select * from paths where time >= startTime and time < endTime
+   * time interval include startTime and exclude endTime
+   * @param paths
+   * @param startTime included
+   * @param endTime excluded
+   * @return
+   * @throws StatementExecutionException
+   * @throws IoTDBConnectionException
+   */
+
+  public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, long endTime)
+          throws StatementExecutionException, IoTDBConnectionException {
+
+    String statement = SessionUtils.rawDataQuery;
```

Comment:
In JDBC or executeQueryStatement in Session, we send a SQL to the server, the SQL is used in a web metric(SqlArgument), which is disabled by default. The SQL is not important.

We could remove the statement field in the rpc. if we need it, we could reconstruct a sql in the server based on other parameters.

Reply:
Got it. Thank you Jialin for your comments!